src/memory/textMemory.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Contains function instructions, as encoding it to the virtual memory text section is too difficult
```

Comment:
I agree, let's work with this for now because to truly have a text section would require us to have a compilation step where the output is bytecode.